how-to/web-interop-support-context-and-intents/client/src/shapes/layout-shapes.ts

```diff
@@ -0,0 +1,43 @@
+import type OpenFin from "@openfin/core";
+import type { WebLayoutSnapshot } from "@openfin/core-web";
+
+/**
+ * Type of the parameter passed into the layoutManagerOverride function.
+ * Extended by implementor's class.
+ */
+export type LayoutManagerConstructor = OpenFin.LayoutManagerConstructor<OpenFin.LayoutSnapshot>;
```

Comment:
You probably want `WebLayoutInitOptions` which changes the `layoutManagerOverride` property to type `OpenFin.LayoutManagerOverride<WebLayoutSnapshot>`

Reply:
Actually if you are using the `fin` type from core, `fin.Platform.Layout.init` would not overlap correctly

Reply:
Will leave as is.